### DISCUSSION TITLE
(FIXED) jplephem excerpt for bce time

### DISCUSSION THREAD

**Discussion by 506f6f68** [skyfielders/python-skyfield#591] [Q&A]
I'm trying to make an excerpt from de441 since the file is so big. My start time and end time are 4BCE/01/01 to 4BCE/12/31. I tried adding negative signs before the year (ex:-0004/01/01) or bc  (ex B.C 0004/01/01), but none of them worked. It all returned an error: argument start_date: specify each date as YYYY or YYYY/MM or YYYY/MM/DD. Any suggestions?

**Comment by brandon-rhodes** [comment]
Unix command line options begin with a dash `-` so Python's argument parser thinks `-0004/01/01` is the series of options `0`, `4`, and so forth. Try this:
```
$ python -m jplephem excerpt -- -4/1/1 -4/12/31 ...
```
The `--` is the Unix way of saying "I'm done with options, everything past here is an argument, even if it starts with a dash."

If you find it works, let me know, and I'll update the documentation to explain that case!

**Edit:** Also, 4 BCE is the year -3 because astronomers have a year zero.

**Reply by 506f6f68** [reply]
It worked! Thank you so much!
